src/librustc_llvm/ffi.rs

```diff
@@ -451,6 +451,7 @@ pub mod debuginfo {
         FlagIndirectVariable = 1 << 13,
         FlagLValueReference = 1 << 14,
         FlagRValueReference = 1 << 15,
+        FlagMainSubprogram = 1 << 21,
```

Comment:
I looks like older LLVM versions will just silently ignore this. At least 3.7 on Travis. Which is good.